### DISCUSSION TITLE
Why doesn't it support redis auth?

### DISCUSSION THREAD

**Discussion by jjangga0214** [Mailu/Mailu#3069] [General]
Hi!

It seems Mailu does not support using a password for Redis.

Is there a reason?

Thanks :)

**Comment by nextgens** [comment]
In Mailu's threat model, there is a clear distinction in between what's inside and outside with regards to the "front" container.

Redis is only one of the many protocols in use "inside", most of them currently do not use any authentication.

While we could add/enable authentication, it is unclear what goal would be achieved by doing so. There is clearly an argument for restricting access to specific databases for specific containers... but that is probably not what you meant.

**Reply by jjangga0214** [reply]
Thank you for the prompt reply!

I fully understand the answer :)

Well, however, I am still skeptical.
(I am not saying mailu must use password.
But I say it should provide an option.)

> Redis is ... "inside", ... do not use any authentication

For example, Postgres is "inside" as well. 
But everyone (including mailu) uses a password.
Why?

**Reply by nextgens** [reply]
I believe that until recently you could not use pgsql without authentication... That would be the main reason.

The other one is that we only use pgsql in admin (and optionally webmail)... Whereas redis is used by more containers.

If we were to add authentication, we would want one set of credentials per db... That would mean many different credentials on admin and more complex setup instructions
